app/views/add-reduction.html

```diff
@@ -0,0 +1,156 @@
+{% extends "includes/layout.html" %} {% block content %}
+
+<main id="content" role="main">
+  {% include "includes/breadcrumbs.html" %} 
+  
+  <header class="heading-xlarge" style="margin-top:0; margin-bottom:0">
+    New reduction
+  </header>
+  {%if failureText %}
```

Comment:
Minor: Formatting - include a bit more whitespace to split out sections?

Reply:
👍